devito/finite_differences/derivative.py

```diff
@@ -178,6 +177,10 @@ def side(self):
     def transpose(self):
         return self._transpose
 
+    @property
+    def is_TimeDependent(self):
```

Comment:
this should be `is_time_dependent`. The camel caps notation is for class attributes denoting a hierarchy